src/Akka.Persistence.Redis.Hosting/RedisJournalOptions.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System.Text;
+using Akka.Configuration;
+using Akka.Hosting;
+using Akka.Persistence.Hosting;
+
+#nullable enable
+namespace Akka.Persistence.Redis.Hosting;
+
+public class RedisJournalOptions : JournalOptions
```

Comment:
The redis journal and snapshot options are separate classes but set exactly the same properties in different hocon sections.